src/Template/Element/sentences/sentence_and_translations.ctp

```diff
@@ -46,10 +66,14 @@ $sentenceUrl = $this->Url->build([
     'action' => 'show'
 ]);
 ?>
-<div ng-cloak
+<div ng-cloak flex
      sentence-and-translations
-     ng-init="vm.init(<?= $userLanguagesJSON ?>, <?= $sentenceJSON ?>, <?= $directTranslationsJSON ?>, <?= $indirectTranslationsJSON ?>)"
+     ng-init="vm.init(<?= $userLanguagesData ?>, <?= $sentenceData ?>, <?= $directTranslationsData ?>, <?= $indirectTranslationsData ?>)"
      class="sentence-and-translations md-whiteframe-1dp">
+    <div ng-if="vm.sentence.duplicate" layout="row" layout-padding style="border-bottom: 1px solid #f1f1f1">
```

Comment:
Maybe put this style in the CSS, otherwise it will be duplicated for each sentence block.